news/6789.bugfix.rst

```diff
@@ -0,0 +1 @@
+Prevent collection of an entire Python site for single-file modules
```

Comment:
Prevent collection of an entire Python site when using ``collect_data_files`` against single-file modules.

Reply:
Or you can really overcompensate and make them cross references ``:cmdoption:`--collect-data` `` or ``:func:`~PyInstaller.utils.hooks.collect_data_files` ``.